tests/compiler/std/symbol.optimized.wat

```diff
@@ -638,29 +640,34 @@
     br $while-continue|0
    end
   end
-  local.get $4
-  local.tee $2
+  local.get $5
+  local.tee $3
   local.get $0
+  local.tee $2
   i32.load
   i32.ne
   drop
-  local.get $0
   local.get $2
+  local.get $3
   i32.store
-  local.get $0
+  local.get $2
   local.get $1
   i32.store offset=4
-  local.get $0
+  local.get $4
+  local.tee $1
+  local.get $2
+  local.tee $3
```

Comment:
Do you have a theory on what's happening here?

Reply:
std/symbol has most significant regress but I can't figure outing how it repair. May be you could try later